spec.bs

```diff
@@ -1421,7 +1421,10 @@ To <dfn>escape a regexp string</dfn> given a string |input|:
 To <dfn export>generate a [=/pattern string=]</dfn> from a given [=/part list=] |part list| and [=/options=] |options|:
 
   1. Let |result| be the empty string.
-  1. [=list/For each=] |part| of |part list|:
+  1. Let |index list| be the result of [=list/getting the indices=] for |part list|.
+  1. [=list/For each=] |index| of |index list|:
+    1. Let |part| be |part list|[|index|].
+    1. Let |last part| be |part list|[|index| - 1] if |index| is greater than 0, otherwise let it be null.
```

Comment:
You could move this under the "Else if full-wildcard"

Reply:
Oh, I see, that would be less nice in combination with #152.

Separate issue though: "last" is confusing; it might mean the last one in the list. I'd suggest "previous"